R/pkg/R/DataFrame.R

```diff
@@ -65,6 +65,49 @@ dataFrame <- function(sdf, isCached = FALSE) {
 
 ############################ SparkDataFrame Methods ##############################################
 
+#' Return an SparkDataFrame's name.
+#'
+#' @param x The SparkDataFrame whose name is returned.
+#' @rdname name
+#' @examples
+#'\dontrun{
+#' sparkR.session()
+#' path <- "path/to/file.json"
+#' df <- read.json(path)
+#' df.name()
+#'}
+#' @aliases name,SparkDataFrame-method
+#' @noRd
+#' @note explain since 2.1.0
```

Comment:
no need for `@noRd` - that's for private function
for `@note` - make sure we have the name of the function instead of `explain`